bokehjs/src/lib/models/sources/columnar_data_source.ts

```diff
@@ -1,17 +1,17 @@
-import {DataSource} from "./data_source"
-import {Signal, Signal0} from "core/signaling"
+import {Geometry} from "core/geometry"
 import {logger} from "core/logging"
-import {SelectionManager} from "core/selection_manager"
 import * as p from "core/properties"
+import {SelectionManager} from "core/selection_manager"
+import {Signal, Signal0} from "core/signaling"
 import {Arrayable, ArrayableNew} from "core/types"
-import {isArray} from "core/util/types"
 import {uniq} from "core/util/array"
+import {is_NDArray} from "core/util/ndarray"
 import {keys, values} from "core/util/object"
-import {Selection} from "../selections/selection"
+import {isArray} from "core/util/types"
+import type {GlyphRenderer} from "../renderers/glyph_renderer"
 import {SelectionPolicy, UnionRenderers} from "../selections/interaction_policy"
-import type {Renderer} from "../renderers/renderer"
-import {Geometry} from "core/geometry"
-import {is_NDArray} from "core/util/ndarray"
+import {Selection} from "../selections/selection"
+import {DataSource} from "./data_source"
```

Comment:
What's going on with those imports?

Reply:
They were sorted, which is mandatory on every other project I work on and also what we should be doing, especially the length (up to dozens) of import lines we have in many cases.

Reply:
But that's not relevant to bokehjs, though such change could be considered. If you want to change that, discuss it first. Then we can establish style rules (perhaps resolve relative vs. absolute imports situation at the same time), set up linting and apply those changes consistently across the codebase at a convenient time. I'm only guessing that unrelated changes like these will continue in new PRs, making merging other PRs even more tedious.

Reply:
>  I'm only guessing that unrelated changes like these will continue in new PRs

That's a good reason to actually define a standard and enforce it via automation. I will happily adopt absolutely any standard that can be configured in the repo to be picked up by vscode. Until there, there is technically and actually no definition of "wrong" here, so I am not going to spend more time trying to juggle things.